safehttp/mux.go

```diff
@@ -210,8 +202,8 @@ func (h handlerWithInterceptors) ServeHTTP(w http.ResponseWriter, r *http.Reques
 		}
 	}()
 
-	for _, key := range h.intercepsOrder {
-		h.interceps[key].Before(rw, ir, h.configs[key])
+	for _, ai := range h.interceps {
```

Comment:
`ai` isn't clear when you see `h.intercepts`. I'd suggest changing to `interceptor`.